docs/extensions/readme.md

```diff
@@ -20,15 +20,17 @@ The following documents describes AutoRest specific vendor extensions for [OpenA
 * [x-ms-error-response](#x-ms-error-response) - indicates whether the response status code should be treated as an error response
 * [x-ms-text](#x-ms-text) - XML helper for scenario with attributes and text on the same XML node
 * [x-ms-client-default](#x-ms-client-default) - Set default value for properties and parameters
+* [x-ms-pageable](#x-ms-pageable) - allows paging through lists of data.
```

Comment:
moved some of these extensions from the `Microsoft Azure Extensions (available in most generators only when using `--azure-arm`)` section. Def could've missed some extensions in that section though